docs/azure/configure-aca-environments.md

```diff
@@ -0,0 +1,70 @@
+- <xref:Aspire.Hosting.AzureContainerAppContainerExtensions.PublishAsAzureContainerApp*?displayProperty=nameWithType>
+- <xref:Aspire.Hosting.AzureContainerAppExecutableExtensions.PublishAsAzureContainerApp*?displayProperty=nameWithType>
+
+These APIs automatically create a default ACA environment when you publish your app. While this default setup works well for most scenarios, you might need to customize the ACA environment to meet specific requirements. To achieve this, use the `AddAzureContainerAppEnvironment` method.
+
+The [.NET Aspire app host](../fundamentals/app-host-overview.md) simplifies infrastructure provisioning by generating code to create Azure resources for your applications. This approach enables you to model and configure deployment-related aspects directly in C#, reducing the need to rely on tools like Bicep. This includes configuring ACA environments, which provide a serverless platform for running containerized applications.
+
+By using the <xref:Azure.Provisioning> APIs (explained in [Infrastructure as code](integrations-overview.md#infrastructure-as-code)), you can configure and customize ACA environments along with related resources, such as container registries and file share volumes. Any available deployment setting can be configured. For more information on the available settings, see [Microsoft.App managedEnvironments](/azure/templates/microsoft.app/managedenvironments).
+
+This article guides you through the process of tailoring ACA environments for your .NET Aspire solutions.
+
+## Add an ACA environment
+
+<!-- TODO: Add xref to AddAzureContainerAppEnvironment when available -->
+
+The `AzureContainerAppEnvironmentResource` type models an ACA environment resource. When you call the `AddAzureContainerAppEnvironment` method, it creates an instance of this type (wrapped in the <xref:Aspire.Hosting.ApplicationModel.IResourceBuilder`1>).
+
+:::code language="csharp" source="snippets/AspireAca.AppHost/AspireApp.AppHost/Program.cs":::
+
+By default, the calling this API to add an ACA environment generates the following provisioning Bicep module:
+
+:::code language="bicep" source="snippets/AspireAca.AppHost/AspireApp.AppHost/aca-env.module.bicep":::
+
+This module configures:
+
+- A user-assigned managed identity for the ACA environment.
+- An Azure Container Registry (ACR) for the ACA environment.
+- A Log Analytics workspace for the ACA environment.
+- An Azure Container Apps environment.
+- The [.NET Aspire dashboard](../fundamentals/dashboard/overview.md) for the ACA environment.
+- A role assignment for the user principal ID to the ACA environment.
+- Various outputs for the ACA environment.
+
+Using the `acaEnv` variable, you can chain a call to the <xref:Aspire.Hosting.AzureProvisioningResourceExtensions.ConfigureInfrastructure*> API to customize the ACA environment to your liking. For more information, see [Configure infrastructure](integrations-overview.md#configure-infrastructure).
```

Comment:
Do we want to add a very simple example snippet here?

The first time I skimmed over the doc, I missed this line. It doesn't really stand out, like a code block would.

Reply:
Yes, I like that idea - I'll add it first thing tomorrow morning.